gammapy/maps/tests/test_base.py

```diff
@@ -7,7 +7,7 @@
 from astropy.coordinates import SkyCoord
 import astropy.units as u
 from astropy.units import Unit, Quantity
-from ...utils.testing import requires_dependency
+from ...utils.testing import requires_dependency, assert_quantity_allclose
```

Comment:
Suggest to use explicit assert on units and values of the results:
https://docs.gammapy.org/0.8/development/howto.html#assert-convention

Reply:
OK

Reply:
I have kept some `assert_quantity_allclose` because for multiplication and division the units can become a bit complex.

Reply:
But if you choose "m" and "s" for units, or make one dimensionless "", it doesn't get very complicated. Note that Unit objects compare with strings:
```
>>> u.Unit('m s') == 'm s'
True
```
so in tests to assert units you can just write:
```
assert m.unit == 'm s'
```
for the product of "m" and "s".

Reply:
Done